### PR TITLE
design changes for question page

### LINKED ISSUE #6294: Create new question page (following style guide!)
#1994 describes the new design style guide, and we now have new designs for the basic Note and Wiki pages. But the Questions page is still looking a bit old. We should make the questions page use the same template as the notes page, so it is similarly cleaned up. We may need to preserve the "related questions" on the sidebar. But otherwise let's get it looking nice and minimal again! 

We'd love help with this!

### PR DESCRIPTION
**Pull request by Uzay-G**
Fixes #6294
@pydevsg @lohitha02 @publiclab/reviewers 

Hey! I've integrated the minor changes so the question page will now looks like this:

question-design

I'd be happy to change anything else if you have any ideas?
Also I was wondering, for the "x answers" part in the about question section, do we display the number of comments or is there another more precise way to get the actual number of answers on a question?
Thanks!

### CODE REVIEW

**Comment by SidharthBansal** [comment]
Great work!
Let's wait for @jywarren to review and merge.
I have approved it on the dashboard so that you can work on other issues.
Don't forget to complete this issue before GCI deadline for marks for this pr
Thanks
Sidharth

**Comment by pydevsg** [comment]
Hi, @Uzay-G you have done a great work, please share a screenshot asap to get points for the task.

**Comment by Uzay-G** [comment]
Thanks @pydevsg! I put a screenshot in the body of the pr.

**Comment by pydevsg** [comment]
Okay, great you can also share the screenshot in this comment so that we can look upon them.
Thanks

**Comment by Uzay-G** [comment]
Yeah here you go:
question-design

**Comment by pydevsg** [comment]
Excellent :clap: 
cc @jywarren

**Comment by SidharthBansal** [comment]
LGTM :rocket:
cc/ @jywarren

**Comment by jywarren** [comment]
Hi @Uzay-G - can you point out exactly what the changes are that you made, visually? I'm having trouble picking them out when compared to the existing page at https://publiclab.org/questions/adam-griffith/08-27-2019/favorite-camera-for-balloon-kite-mapping

Thank you!

File: app/views/questions/show.html.erb [hunk 1]
```diff
@@ -24,7 +23,7 @@
   </div>
   <div class="pull-left">
   <% if @node.author && @node.author.photo_file_name %>
-    <img class="rounded-circle" src="<%= @node.author.photo_path(:thumb) %>" style="width:50px;margin-right:10px;" />
+    <img class="rounded-circle" src="<%= @node.author.photo_path(:thumb) %>" style="width:150px;margin-right:10px;" />
```
Comment (jywarren) [inline app/views/questions/show.html.erb]: Ah, we had actually reverted this back to 50, as it had been too big! Maybe you've mistaken the screenshot of how it WAS with the screenshot of how we want it to be?

**Comment by Uzay-G** [comment]
I followed the model of this image you suggested:
https://user-images.githubusercontent.com/24359/65049912-900feb80-d934-11e9-8995-2a485774bd02.png.
The image profile was made bigger and I added the answers part with the comment icon. I also made some minor edits to make everything responsive. The changes in the image you suggested are pretty minor but I put into practice what it suggested.

**Comment by Uzay-G** [comment]
Oh! I think I did that mistake.  I got confused because the way the page looks on publiclab at the moment was very different from the image you linked at the time.

**Comment by Uzay-G** [comment]
I am so sorry @jywarren. I made a mistake but I fixed it. Is this what you had in mind for the question page:

**Comment by Uzay-G** [comment]
I changed the files accordingly

**Comment by jywarren** [comment]
Hi, this looks good; the new Notes pages are even more minimal in the metadata under the title; do you think you could compress it further? I think a substantial amount of the info is already now in the `...` menu, as we moved it there for the notes views and the menu is shared code. What do you think?

**Comment by Uzay-G** [comment]
Something more like this? 
question-design

**Comment by SidharthBansal** [comment]
@jywarren kindly review it. Thanks

**Comment by SidharthBansal** [comment]
fantastic

**Pull request merged**